<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>641</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>OBSHadowplay.Settings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="exceptions_label">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>OBSHadowplay.Settings.Exclusions</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListWidget" name="exceptions_list">
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="add_exception_button">
           <property name="toolTip">
            <string>OBSHadowplay.Settings.AddExclusion</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="toolButton" stdset="0">
            <bool>true</bool>
           </property>
           <property name="class" stdset="0">
            <string notr="true">icon-plus</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="edit_exception_button">
           <property name="toolTip">
            <string>OBSHadowplay.Settings.EditExclusion</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="toolButton" stdset="0">
            <bool>true</bool>
           </property>
           <property name="class" stdset="0">
            <string notr="true">icon-cogs</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="delete_exception_button">
           <property name="toolTip">
            <string>OBSHadowplay.Settings.DeleteExclusion</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="toolButton" stdset="0">
            <bool>true</bool>
           </property>
           <property name="class" stdset="0">
            <string notr="true">icon-trash</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="group_box_buffer">
     <property name="toolTip">
      <string>OBSHadowplay.Settings.ReplayBuffer.EnableAutomaticStartStop</string>
     </property>
     <property name="title">
      <string>OBSHadowplay.Settings.ReplayBuffer</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,1" columnminimumwidth="100,0">
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>OBSHadowplay.Settings.ReplayBuffer.StopDelay</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QSpinBox" name="buffer_stop_delay_spinbox">
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="reset_buffer_on_save_label">
        <property name="text">
         <string>OBSHadowplay.Settings.ReplayBuffer.ResetOnSave</string>
        </property>
        <property name="class" stdset="0">
         <string>icon-trash</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="reset_buffer_on_save_checkbox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="reset_on_save_info_icon">
          <property name="toolTip">
           <string>OBSHadowplay.Settings.ReplayBuffer.ResetOnSave.Warning</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_box_organisation">
     <property name="toolTip">
      <string>OBSHadowplay.Settings.Organisation.EnableAutoOrganisation</string>
     </property>
     <property name="title">
      <string>OBSHadowplay.Settings.Organisation</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1" columnminimumwidth="100,0">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <item row="1" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>OBSHadowplay.Settings.Organisation.IncludeScreenshots</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QGroupBox" name="custom_filename_groupbox">
        <property name="toolTip">
         <string>OBSHadowplay.Settings.Organisation.CustomFilename.EnableCustomFilename</string>
        </property>
        <property name="title">
         <string>OBSHadowplay.Settings.Organisation.CustomFilename</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_4" columnminimumwidth="90,0">
         <item row="1" column="0">
          <widget class="QComboBox" name="custom_filename_format_arrangement">
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="currentIndex">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLineEdit" name="custom_filename_separator_textbox">
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>OBSHadowplay.Settings.Organisation.CustomFilename.Separator</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>OBSHadowplay.Settings.Organisation.CustomFilename.Arrangement</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="include_screenshots_checkbox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="folder_organisation_checkbox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>OBSHadowplay.Settings.Organisation.EnableFolderOrganisation</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_box_notifications">
     <property name="title">
      <string>OBSHadowplay.Settings.Notifications</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,1" columnminimumwidth="100,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item row="1" column="0">
       <widget class="QCheckBox" name="show_desktop_notif_checkbox">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>OBSHadowplay.Settings.Notifications.ShowDesktopNotification</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="play_notification_sound_checkbox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>OBSHadowplay.Settings.Notifications.PlayNotificationSound</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
